packages/client/hmi-client/src/components/llm/tera-jupyter-chat.vue

```diff
@@ -8,6 +8,10 @@
 			tabindex="0"
 			class="message-container"
 		>
+			<div v-if="showRerunMessage" class="rerun-message">
+				Re-run all the cells to restore the context if you need to make any changes or use them downstream.
+				<Button class="close-mask" icon="pi pi-times" text rounded aria-label="Close" @click="hideRerunMessage" />
+			</div>
```

Comment:
It will be fun if clicking on the banner did the same thing.
```suggestion
			<div v-if="showRerunMessage" class="rerun-message"  @click="window.dispatchEvent(new Event(EventType.RunAllCells))">
				Re-run all the cells to restore the context if you need to make any changes or use them downstream.
				<Button class="close-mask" icon="pi pi-times" text rounded aria-label="Close" @click.stop="hideRerunMessage" />
			</div>
```